core/src/retransmit_stage.rs

```diff
@@ -254,6 +262,7 @@ fn retransmit(
         w_epoch_stakes_cache.stakes_and_index = stakes_and_index;
         drop(w_epoch_stakes_cache);
         r_epoch_stakes_cache = epoch_stakes_cache.read().unwrap();
+        shreds_received.lock().unwrap().clear();
```

Comment:
Wondering if we should change `now - last > 1000` here: https://github.com/solana-labs/solana/pull/14030/files#diff-8d73ca7d46b0b2dba59642e0e183fbb6127c8459ade483df4ac729390519e6f6R255 to `now - last > NUM_CONSECUTIVE_LEADER_SLOTS * DEFAULT_MS_PER_SLOT` because that's how often the propagation check may retransmit.

Reply:
Those timings aren't exact, it just needs to prevent it from creating a huge amount of unrestricted traffic.